### DISCUSSION TITLE
How is everyone storing ISO addresses for ISIS interfaces?

### DISCUSSION THREAD

**Discussion by bradbendy** [netbox-community/netbox#13467] [General]
Are you guys doing a custom field under  either the interface or the IPv4/IPv6 address fields? Not sure the best to track this, would be nice to have it show up under the interfaces with the IP addresses but don't think that will be possible. Or is there another way I have never seen?

Thanks

**Comment by candlerb** [comment]
Netbox doesn't natively handle NSAP (NET) addresses, so you won't see these included with the IPv4 and IPv6 addresses.

However, a custom field on Interface would show up on the Interfaces listing (you can control which columns are shown). That to me seems like the logical place for it, and is simple assuming you don't ever want multiple NSAP addresses on the same interface - which I don't see why you'd ever want.

**Comment by bradbendy** [comment]
Yeah custom field on the interface does the trick, didn't realize you could add custom fields to the layout, which is perfect! Thanks!
